static/js/autocomplete.js

```diff
@@ -0,0 +1,9 @@
+var autocompleteElements = document.getElementsByName('autocompleteElements')[0].content.split(',')
```

Comment:
I've tried to simplify all of these autocomplete scripts and reduce duplication by passing the element in as an argument by using meta tags